kerastuner/engine/hyperparameters.py

```diff
@@ -133,22 +146,34 @@ class Int(HyperParameter):
         min_value: Int. Lower limit of range (included).
         max_value: Int. Upper limit of range (excluded).
         step: Int. Step of range.
+        sampling: Optional. One of "linear", "log",
+            "reverse_log". Acts as a hint for an initial prior
+            probability distribution for how this value should
+            be sampled, e.g. "log" will assign equal
+            probabilities to each order of magnitude range.
         default: Default value to return for the parameter.
             If unspecified, the default value will be
             `min_value`.
     """
 
-    def __init__(self, name, min_value, max_value, step=1, default=None):
+    def __init__(self,
+                 name,
+                 min_value,
+                 max_value,
+                 step=1,
+                 sampling=None,
+                 default=None):
         super(Int, self).__init__(name=name, default=default)
         self.max_value = int(max_value)
         self.min_value = int(min_value)
         self.step = int(step)
```

Comment:
Any reason to silently cast to int and not raise an error if step is not an integer?

Reply:
Not sure, other than we probably want to allow `1.0`. We could check `int(step) == step` maybe

@fchollet for context

Reply:
int(step) == step SGTM.

Reply:
Done